CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_salesforce_formula_utils v0.9.1
+## Bugfix
+- Add Databricks to a conditional in the `sfdc_formula_view_sql` macro so backticks correctly compile in said warehouse. ([#94](https://github.com/fivetran/dbt_salesforce_formula_utils/pull/94))
```

Comment:
This reads a bit strange. Can you adjust this to be clearer as to why this fix was needed and what solution was applied. 

Also, it would be worthwhile to include the customers PR reference next to this one to show their contribution.

Reply:
Updated